Zend/zend_gdb.c

```diff
@@ -105,6 +110,7 @@ ZEND_API void zend_gdb_unregister_all(void)
 ZEND_API bool zend_gdb_present(void)
 {
 	bool ret = 0;
+#if defined(__linux__) /* netbsd while having this procfs part, does not hold the tracer pid */
```

Comment:
Wouldn't it better to
```suggestion
#ifndef __NetBSD__ /* netbsd while having this procfs part, does not hold the tracer pid */
```
Just to avoid issue with other systems.

Or is this relevant for other BSD systems as well?

Reply:
this is a code path only valid on linux.

Reply:
Thanks for clarification!